pkg/cli/config/config.go

```diff
@@ -85,6 +98,21 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 	return bpConfig, nil
 }
 
+// GetBackplaneURL returns API URL
+func (config *BackplaneConfiguration) GetBackplaneURL() (string, error) {
+
+	ocmEnv, err := ocm.DefaultOCMInterface.GetOCMEnvironment()
+	if err != nil {
+		return "", err
+	}
+	url, ok := ocmEnv.GetBackplaneURL()
+	if !ok {
+		return "", fmt.Errorf("no backplane API defined for %v", ocmEnv.Name())
```

Comment:
nit

```suggestion
		return "", fmt.Errorf("The requested API endpoint is not available for the OCM environment: %v", ocmEnv.Name())
```

Reply:
Will change it to `the` since 
```
pkg/cli/config/config.go:108:14: ST1005: error strings should not be capitalized (stylecheck)
                return "", fmt.Errorf("The requested API endpoint is not available for the OCM environment: %v", ocmEnv.Name())
```